apps/lukso-manager/src/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"lukso/apps/lukso-manager/src/downloader"
+	"lukso/apps/lukso-manager/src/metrics"
+	"lukso/apps/lukso-manager/src/runner"
+	"lukso/apps/lukso-manager/src/validator"
+	"lukso/apps/lukso-manager/src/webserver"
+
+	"github.com/gorilla/mux"
+)
+
+func main() {
```

Comment:
do `/health` as well, so we remove this exec command from gke

Reply:
Good Idea.

- [ ] Add GET /health endpoint